src/Foundation/NSUrlSessionHandler.cs

```diff
@@ -168,7 +168,9 @@ public NSUrlSessionHandler (NSUrlSessionConfiguration configuration)
 				configuration.TLSMinimumSupportedProtocol = SslProtocol.Tls_1_1;
 			else if ((sp & SecurityProtocolType.Tls12) != 0)
 				configuration.TLSMinimumSupportedProtocol = SslProtocol.Tls_1_2;
-
+			else if ((sp &  SecurityProtocolType.Tls13) != 0)
+				configuration.TLSMinimumSupportedProtocol = SslProtocol.Tls_1_3;
```

Comment:
The [mono PR](https://github.com/mono/mono/pull/14378] you're referencing updates `SecurityProtocolType` (from `Mono.Security`) and not `SecurityProtocolType` (from `System.Net`). 

Those are different types for historical reasons: the old Mono managed SSL implementation was based on, never released, .NET 1.2 betas.

Reply:
However this does not _really_ need a mono update to work. You can type cast `12288` into a `SecurityProtocolType`